### DISCUSSION TITLE
Can't search VoucherFileConnections by year

### DISCUSSION THREAD

**Discussion by SWICRIOT** [FortnoxAB/csharp-api-sdk#293] [Q&A] [answered]
```
var generalSearch = new Fortnox.SDK.Search.VoucherFileConnectionSearch
{
    Limit = 20,
    FinancialYearID = vouchers.First().Year
};

var voucherFiles = await voucherFileConnectionConnector.FindAsync(generalSearch);
```

This is returning VoucherFileConnections with mixed financial years. I can't see what else I'm supposed to do.

**Comment by adamelfstrom** [comment, marked as answer]
Hi, 
`FinancialYearID` is a part of the BaseSearch parameters and does not have an impact on VoucherFileConnectionSearch.

The correct parameter should be `VoucherYear`, as seen here https://apps.fortnox.se/apidocs#tag/VoucherFileConnections
Although it seems this parameter has not been added to this SDK yet.

FYI
@richardrandak

**Reply by richardrandak** [reply]
Cool to see they have better documentation now :) too bad i can't actively maintain the SDK anymore..
However, I believe there is option to add custom parameters in the search object, so even though the SDK do not have it explicitely defined, you can use them

**Reply by SWICRIOT** [reply]
Thanks. Maybe the API would be more clear if the search classes would not contain properties that are not relevant.
